### PR TITLE
Fix issue that init routine runs in global scope

### PR DESCRIPTION
**Pull request by usami-k**
Fix issue that init routine runs in global scope. (Especially, setopt influences global environment.)

### CODE REVIEW

**Comment by b4b4r07** [comment]
Thanks but anonymous function is only zsh feature.

**Comment by usami-k** [comment]
> Thanks but anonymous function is only zsh feature.

Oh, sorry.
Rewrite pull request to use a named function.

File: init.sh [hunk 1]
```diff
@@ -67,3 +68,5 @@ fi
         setopt localoptions SH_WORD_SPLIT
     fi
 } &>/dev/null
```
Comment (b4b4r07) [inline init.sh]: It feels like this `/dev/null` is unnecessary

**Comment by usami-k** [comment]
Remove redirection. Thanks for your review.

**Comment by b4b4r07** [comment]
Thank you!

**Pull request merged**